apps/cli/src/compose/rollups/docker-compose-espresso.yaml

```diff
@@ -19,7 +19,7 @@ services:
             ESPRESSO_BUILDER_PORT: 23000
             ESPRESSO_DEV_NODE_PORT: 20000
             ESPRESSO_SEQUENCER_ETH_MNEMONIC: ${CARTESI_AUTH_MNEMONIC:-test test test test test test test test test test test junk}
-            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "200ms"
+            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "5s"
```

Comment:
how about we use the same BLOCK_TIME env variable we are injecting in the compose environment for the anvil service?
```
${BLOCK_TIME:-5}
```
I don't think it will make much of a difference to poll with the same frequency and the anvil node block time.
If the user wants a shorter one he can still change it in the CLI.

Reply:
I think we dropped the support for injecting env variables from `.cartesi.env` when we introduced the `rollups start` command.

Reply:
Maybe we should have kept it, and it was removed by mistake, not sure.

Reply:
I don’t mean the env file. We already have a block-time command option

Reply:
sent a fixup at 7f014f9